osquery/tables/system/windows/os_version.cpp

```diff
@@ -44,6 +157,23 @@ QueryData genOSVersion(QueryContext& context) {
   default:
     break;
   }
+
+  r["platform"] = "windows";
+  r["platform_like"] = "windows";
+  r["version"] = r["major"] + "." + r["minor"] + "." + r["build"];
+  if (version.size() >= 2) {
```

Comment:
I think this check is really redundant. I only dropped it to be safe and ensure the call to `GetProductInfo` has values. From what I can tell, every version of windows will _always_ have a major, minor, and build number.